### DISCUSSION TITLE
Rename the API’s methods to be more meaningful

### DISCUSSION THREAD

**Discussion by duckydash139** [solidjs/solid-start#294] [Ideas]
The current form are something like

get, post, put, patch, del

imo, it's pretty generic and it takes time to recognize.

So i would like to purpose to rename it like

onRequestGet
onRequestDelete
..and so on

What do you think?

**Comment by nksaraf** [comment]
We changed it to `GET`, `POST`, `PATCH`. This makes it look more like the http method names that they are handling and that makes it stand out. The longer `onRequestGet` doesn't add too much recognizability and more characters.
